homeassistant/components/aidot/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""The aidot integration."""
+
+from __future__ import annotations
+
+from collections.abc import Mapping
+import logging
+from typing import Any
+
+from aidot.const import (
+    CONF_DEVICE_LIST,
+    CONF_ID,
+    CONF_LOGIN_RESPONSE,
+    CONF_PRODUCT_LIST,
+)
+from aidot.discover import Discover
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError
+from homeassistant.helpers import device_registry as dr
+
+from .coordinator import AidotConfigEntry, AidotCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.LIGHT]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: AidotConfigEntry) -> bool:
+    """Set up aidot from a config entry."""
+
+    coordinator = AidotCoordinator(
+        hass,
+        entry,
+        entry.data[CONF_DEVICE_LIST],
+        entry.data[CONF_LOGIN_RESPONSE],
+        entry.data[CONF_PRODUCT_LIST],
+    )
+    entry.runtime_data = coordinator
```

Comment:
```suggestion
    await coordinator.async_config_entry_first_refresh()
    entry.runtime_data = coordinator
```
refresh the coordinator once